packages/firestore/test/lite/integration.test.ts

```diff
@@ -106,14 +106,56 @@ describe('Firestore', () => {
     expect(fs1).to.be.an.instanceOf(Firestore);
   });
 
-  it('returns same instance', () => {
+  it('returns same default instance from named app', () => {
```

Comment:
Request to add:

* Test some real actions (queries) for the created dbs.
* Create a db with 'name1' and defaultApp, write some document and terminate it. Create another db with 'name1' and explicitly specifies defaultApp, make sure you can read the previous document from cache. Ideally also a db with 'name2' and make sure data is not leaking.

Reply:
Done.